### PR TITLE
Remove stuff

### PR DESCRIPTION
**Pull request by jd-lara**
Removes unused chunks of code and substitutes JSON for JSON2

### CODE REVIEW

File: src/parsers/forecast_parser.jl [hunk 1]
```diff
@@ -254,7 +254,7 @@ function make_forecast_array(all_components, parsed_forecasts::Vector{Dict})
 function write_to_json(filename,Forecasts_dict)
     for (type_key,type_fc) in Forecasts_dict
         for (device_key,device_dicts) in type_fc
-            stringdata =JSON.json(device_dicts, 3)
+            stringdata =JSON2.json(device_dicts, 3)
```
Comment (daniel-thom) [inline src/parsers/forecast_parser.jl]: Does this work?  I don't think JSON2 has the same functions.

Comment (jd-lara) [reply src/parsers/forecast_parser.jl]: It passed all the tests

File: src/parsers/pm_io/data.jl [hunk 2]
```diff
@@ -61,9 +61,10 @@ function calc_theta_delta_bounds(data::Dict{String,Any})
     end
 
     if haskey(data, "conductors")
-        amin = MultiConductorVector(angle_min)
-        amax = MultiConductorVector(angle_max)
-        return amin, amax
+        @error("Multiconductor Not Supported in PowerSystems")
```
Comment (daniel-thom) [inline src/parsers/pm_io/data.jl]: This makes the function return nothing.  The calling code will probably fail with a strange error message.  Do you want to use error() instead?

File: src/parsers/forecast_parser.jl (hunk 1 above)
Comment (daniel-thom) [reply src/parsers/forecast_parser.jl]: I don't believe you.  :)

Comment (daniel-thom) [reply src/parsers/forecast_parser.jl]: Perhaps those functions don't get called.

File: src/parsers/pm_io/data.jl [hunk 3]
```diff
@@ -611,7 +611,7 @@ function check_thermal_limits(data::Dict{String,Any})
     for branch in branches
         if !haskey(branch, "rate_a")
             if haskey(data, "conductors")
-                @error("Multiconductor Not Supported in PowerSystems")
+                error("Multiconductor Not Supported in PowerSystems")
                 #branch["rate_a"] = MultiConductorVector(0.0, data["conductors"])
```
Comment (daniel-thom) [inline src/parsers/pm_io/data.jl]: Is there a reason to keep the commented-out code around?

Comment (jd-lara) [reply src/parsers/pm_io/data.jl]: Just in case we want it for reference later. But we can delete it too.

File: src/parsers/forecast_parser.jl [hunk 4]
```diff
@@ -254,7 +254,7 @@ function make_forecast_array(all_components, parsed_forecasts::Vector{Dict})
 function write_to_json(filename,Forecasts_dict)
     for (type_key,type_fc) in Forecasts_dict
         for (device_key,device_dicts) in type_fc
-            stringdata =JSON2.json(device_dicts, 3)
+            stringdata =JSON2.write(device_dicts, 3)
```
Comment (daniel-thom) [inline src/parsers/forecast_parser.jl]: I still don't think this is correct.  It should fail with the 3.  That being said, this function and the next can likely be deleted.

Comment (jd-lara) [reply src/parsers/forecast_parser.jl]: image

Comment (daniel-thom) [reply src/parsers/forecast_parser.jl]: The function above it (make_forecast_array) can also be deleted.  I thought I deleted it in a previous PR.

Comment (jd-lara) [reply src/parsers/forecast_parser.jl]: I will leave you to delete these in #266 if you think the forecasting won't need them

Comment (daniel-thom) [reply src/parsers/forecast_parser.jl]: Ok.  But you're still breaking working code.  Just delete ", 3" in both cases.

File: src/parsers/forecast_parser.jl (hunk 1 above)
Comment (jd-lara) [reply src/parsers/forecast_parser.jl]: Probably not, we would be able to see that in the Codecov report

File: src/parsers/json2ps_parser.jl [hunk 5]
```diff
@@ -287,7 +286,7 @@ function json_parser(filename)
         open("../data/CDM/RTS/JSON/RTS-GMLC_Test_Case.json", "r") do f
         global temp
         dicttxt = readstring(f)  # file information to string
-        temp = JSON.parse(dicttxt)  # parse and transform data
+        temp = JSON2.parse(dicttxt)  # parse and transform data
```
Comment (daniel-thom) [inline src/parsers/json2ps_parser.jl]: Did you make sure this still works?

File: src/parsers/pm_io/common.jl [hunk 6]
```diff
@@ -34,7 +34,7 @@ end
 ""
 function parse_json(io::IO; validate=true)
     data_string = read(io, String)
-    pm_data = JSON.parse(data_string)
+    pm_data = JSON2.parse(data_string)
```
Comment (daniel-thom) [inline src/parsers/pm_io/common.jl]: and this

Comment (jd-lara) [reply src/parsers/pm_io/common.jl]: I just ran the tests and everything ran. These lines probably never get called.

Comment (daniel-thom) [reply src/parsers/pm_io/common.jl]: They might be correct. I think you should confirm that it generates the same result before pushing.

Comment (jd-lara) [reply src/parsers/pm_io/common.jl]: You are correct, this doesn't return the same thing.

**Pull request merged**